.travis.yml

```diff
@@ -1,20 +1,20 @@
 language: cpp
 cache:
   - ccache
-dist: trusty
+dist: xenial
 sudo: false
 addons:
   apt:
     sources: &add-sources
-    - sourceline: 'ppa:beineri/opt-qt571-trusty'
-    - sourceline: 'ppa:beineri/opt-qt-5.10.1-trusty'
+    - sourceline: 'ppa:beineri/opt-qt571-xenial'
+    - sourceline: 'ppa:beineri/opt-qt-5.10.1-xenial'
     - sourceline: 'ppa:ubuntu-toolchain-r/test'
     - sourceline: 'ppa:ondrej/php'
     packages: &common-packages
     - libhunspell-dev
     - lua5.1
     - liblua5.1-0-dev
-    - libboost-graph1.55-dev
+    - libboost-graph-dev
     - zlib1g-dbg
     - libzip-dev
```

Comment:
What `libzip` will this pull in nowadays, a version **1.x** one I guess?

Reply:
I've had a look in the logs for you - it is 1.5.1.